Lib/glyphsLib/builder/groups.py

```diff
@@ -87,13 +88,41 @@ def to_ufo_groups(self):
             source.font.groups[name] = glyphs[:]
 
 
+def _to_ufo_groups(glyph, groups, glyph_group, side, other_side):
+    if not glyph_group.endswith(".RTL"):
```

Comment:
what if a user names their leftKerningGroup or rightKerningGroup with .RTL suffix...

Reply:
I don't have a definite answer to this question. But I took the opportunity to replace all remaining instances of `name.replace(".RTL", "")` with `name[:-4]` as it was already used in `groups.py`. This should preserve any such a manually named group. Is this important enough to have me check this specifically?

Reply:
I see what you mean. If such a manually named group comes in here in `groups.py`, it would probably clash with this functionality.

Reply:
We could decide for a more unusual ending such as `.__RTL__`

Reply:
if the suffix is only a temporary placeholder and never gets ends up in the generated sources because we stirp it as soon as we have figured out RTL-ness of a kerning pair, then yes, we definitely need to use one that is as unique as possible and that must not clash with one a user might use.
Still a bit unclear to me why you need to check if the group ends with .RTL or not.. is it to avoid adding it twice? Why could that happen?

Reply:
Two things were previously happening: I got groups that had double .RTL ending and I had groups that had all glyphs twice in it and I had all groups twice too, I believe, once with this side number and the other ending, once with the other side number and this ending. The code gets called twice somewhere, and even if only for the conversion back to Glyphs, don't ask me where. The current implementation prevents that in a reactionary manner.

Other topic: Now I know why I started purging the .RTL ending from all names: by doing that, we cleanly separate the data layer from the application layer, and revert to a clean G2 state. It's really Georg's responsibility to sort that on import which he never did but still should. I'll do that tomorrow

Reply:
Before we're done, I would like to add a paragraph to the main README about this. People need to know exactly what's going on so they understand how to handle UFO->Glyphs kerning, even if that's not the main use case of glyphsLib.